pkgs/top-level/all-packages.nix

```diff
@@ -11433,7 +11420,7 @@ let
 
   eiskaltdcpp = callPackage ../applications/networking/p2p/eiskaltdcpp { lua5 = lua5_1; };
 
-  qemu = callPackage ../applications/virtualization/qemu { };
+  qemu = callPackage ../applications/virtualization/qemu (config.qemu or {});
```

Comment:
No sorry but I agree with @peti here, I thought there was already a config.qemu, not that you added a new one. This is something you can do with `packageOverrides`. Don't add this please.